### PR TITLE
Added unspec-service (ucmc-service rename step 1/2)

### PR DESCRIPTION
**Pull request by pd-dudkiewicz**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-467

### Change description ###

Added unspec charts as a part of renaming ucmc (it's better to start from scratch and cleanup after, so team can continue dev work in the meanwhile).

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/aat/common/unspec/identity.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: unspec
+  namespace: unspec
+spec:
+  type: 0
+  ResourceID: /subscriptions/96c274ce-846d-4e48-89a7-d528432298a7/resourceGroups/managed-identities-aat-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/unspec-aat-mi
```
Comment (adusumillipraveen) [inline k8s/aat/common/unspec/identity.yaml:8]: Did u create this ? This isn't just about renaming in here, the resource should exist.

Comment (pd-dudkiewicz) [reply k8s/aat/common/unspec/identity.yaml]: @adusumillipraveen removed for now

File: k8s/preview/unspec/xui-identity.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: xui
```
Comment (adusumillipraveen) [inline k8s/preview/unspec/xui-identity.yaml:4]: xui ?

File: k8s/demo/common/unspec/unspec-service.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    version: 0.0.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/unspec/service:latest
+    global:
+      environment: demo
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      enableKeyVaults: trueT
```
Comment (timja) [inline k8s/demo/common/unspec/unspec-service.yaml:27]:
```suggestion
      enableKeyVaults: true
```

File: k8s/preview/unspec/xui-identity.yaml (hunk 2 above)
Comment (pd-dudkiewicz) [reply k8s/preview/unspec/xui-identity.yaml]: yeah, we're currently using it on preview

**Pull request merged**